From 18cec2bd43953dd2b770c84e71a979d531c3894e Mon Sep 17 00:00:00 2001
From: Longlong Xia <xialonglong1@huawei.com>
Date: Mon, 7 Nov 2022 02:21:16 +0000
Subject: [PATCH 491/595] [Huawei] x86: config: Fix CRYPTO_LZ4 format error

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187943

----------------------------------------

Remove redundant space in openeuler_defconfig while enable
CONFIG_CRYPTO_LZ4

Fixes: 40d12dbf97d4 ("lz4: add a config for this feature")
Signed-off-by: Xia Longlong <xialonglong1@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index eba512b23638..6da13366b192 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -7854,7 +7854,7 @@ CONFIG_CRYPTO_TWOFISH_AVX_X86_64=m
 CONFIG_CRYPTO_DEFLATE=y
 CONFIG_CRYPTO_LZO=y
 # CONFIG_CRYPTO_842 is not set
- CONFIG_CRYPTO_LZ4=m
+CONFIG_CRYPTO_LZ4=m
 # CONFIG_CRYPTO_LZ4HC is not set
 CONFIG_CRYPTO_ZSTD=y
 
-- 
2.17.1

